package.json

```diff
@@ -38,7 +41,8 @@
     "pretest-php-multisite": "wp-env run composer 'install --no-interaction'",
     "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/$(basename $(pwd))/tests/multisite.xml --verbose'",
     "wp-env": "wp-env",
-    "prepare": "husky install"
+    "prepare": "husky install && npm run copy-vendor-libs",
+    "copy-vendor-libs": "node ./bin/plugin/cli.js copy-vendor-libs"
```

Comment:
Note: Partytown also has a built-in Copy Task command which perhaps could be reused? https://partytown.builder.io/copy-library-files

Reply:
> Partytown also has a built-in Copy Task command which perhaps could be reused?

If we will be having our build system(most probably we will be having that for the ILO module), that wouldn't be required.

Reply:
Yeah, you're probably right.